src/main/java/org/spongepowered/api/plugin/PluginManager.java

```diff
@@ -76,4 +90,53 @@
      */
     boolean isLoaded(String id);
 
+    /**
+     * Gets the id of the {@link Plugin}.
+     *
+     * @return The id
+     */
+    default String getId(Object plugin) {
```

Comment:
Purely for convenience. IMO the less verbose we can make these kind of things the better. Similar concept to the `Sponge` class really.